<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Statistics</class>
 <widget class="QWidget" name="Statistics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>799</width>
    <height>583</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="gamesPlayed">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>170</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="playersPlayingGames">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>210</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="topScore">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>290</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="averageGamesPerPlayer">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>250</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="averageGamePlayer">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>370</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="averageGameAll">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>330</y>
     <width>341</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="exitButton">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>220</y>
     <width>141</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string>Return to Menu</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
